src/Nfts/filterNfts.ts

```diff
@@ -0,0 +1,10 @@
+import {YoroiNft} from '../yoroi-wallets'
+
+export const filterNfts = (nftsSearchTerm: string, nfts: YoroiNft[]): YoroiNft[] => {
```

Comment:
`search: string` or `term: string`, `searchTerm`  would be fine for me, the function name is `filterNfts` so I think nft its implied